data/djangocon-2009/videos/djangocon-2009--django-is-obsolete--but-so-is-eve.json

```diff
@@ -1,29 +1,29 @@
 {
-  "id": 33, 
-  "category": "DjangoCon 2009", 
-  "slug": "djangocon-2009--django-is-obsolete--but-so-is-eve", 
-  "title": "Django is obsolete (but so is everything else)", 
-  "summary": "", 
-  "description": "Django is obsolete (but so is everything else)\n\n  \nPresented by Avi Bryant\n\n** Keynote for DjangoCon 2009.**\n\n", 
-  "quality_notes": "", 
-  "language": "English", 
-  "copyright_text": "Creative Commons Attribution-NonCommercial-ShareAlike 3.0", 
-  "thumbnail_url": "http://a.images.blip.tv/Robertlofthouse-DjangoIsObsoleteButSoIsEverythingElse277.png", 
-  "duration": null, 
+  "id": 33,
+  "category": "DjangoCon 2009",
+  "slug": "djangocon-2009--django-is-obsolete--but-so-is-eve",
+  "title": "Django is obsolete (but so is everything else)",
+  "summary": "",
+  "description": "Django is obsolete (but so is everything else)\n\nPresented by Avi Bryant\n\n\\*\\* Keynote for DjangoCon 2009.\\*\\*\n",
```

Comment:
`\*\* Keynote for DjangoCon 2009.\*\*` should probably be `**Keynote for DjangoCon 2009.**` (strong emphasis)